lib/rules/string-no-newline/README.md

```diff
@@ -70,3 +70,33 @@ a {
   font-family: "Times New Roman";
 }
 ```
+
+## Optional secondary options
+
+### `ignore: ["at-rules", "declarations"]`
+
+#### `"at-rules"`
+
+Ignore strings in at-rules.
```

Comment:
minor
pedantic

I don't have a proposition, but based on the example

```css
@import url('example
.com');
```

the current description could be interpreted differently.
i.e. I would expect _in_ in "[…] **in** at-rules." to refer to the scope of a block of a [block at-rule](https://www.w3.org/TR/css-syntax-3/#ref-for-block-at-rule).